src/main/java/com/sap/cloudfoundry/client/facade/CloudControllerClientImpl.java

```diff
@@ -121,8 +121,8 @@ public String createServiceBroker(CloudServiceBroker serviceBroker) {
     }
 
     @Override
-    public CloudServiceKey createServiceKey(String serviceInstanceName, String serviceKeyName, Map<String, Object> parameters) {
-        return handleExceptions(() -> delegate.createServiceKey(serviceInstanceName, serviceKeyName, parameters));
+    public void createServiceKey(String serviceInstanceName, String serviceKeyName, Map<String, Object> parameters) {
```

Comment:
Creating a service credential binding to a managed service instance (https://v3-apidocs.cloudfoundry.org/version/3.119.0/#create-a-service-credential-binding) does not return an object, only a job id. 
And since service keys can't be created for user-provided services, the return param is changed